cmd/fleetctl/apply_test.go

```diff
@@ -1632,6 +1652,34 @@ spec:
 	assert.Equal(t, "select * from app_schemes;", appliedQueries[0].Query)
 }
 
+func SetupDEPStorageAndDEPServer(t *testing.T) *nanodep_mock.Storage {
```

Comment:
since this is not part of a test package, can we make it explicit in the name that it's a mock?